Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Coupled-Transmons/Use Case 4 - Single- and Two-Qubit State and Process Tomography/README.md

```diff
@@ -0,0 +1,185 @@
+# Single- and Two-Qubit State and Process Tomography
+
+_Authors and experimenters: Nicholas Bornman, Paul Heidler and Ziwen Huang_
+
+*Important note: the code in this folder is the code that was used for running these measurements on a pair of flux-tunable*
+*qubits coupled with a flux-tunable coupler. The code may need to be modified for a different piece of hardware*
+*and software environment.*
+
+This use-case has four potential applications, some of which may not be applicable/necessary, namely
+1. Single-qubit state tomography,
+2. Single-qubit process tomography,
+3. Two-qubit state tomography, and
+4. Two-qubit process tomography
+
+The most complex applications are obviously the two-qubit cases, which is what we will focus on below.
+The single-qubit cases follow trivially from them.
+
+## 1. Experimental set-up
+
+<img align="right" src="setup.png" alt="setup" width="400"/>
+
+
+### 1.1 Experimental set-up
+The chip consists of 9 2D flux-tunable transmons arranged into a 3x3 grid, with nearest neighbours connected via flux tunable lines.
+
+For the purposes of this experiment, we focus on two neighbouring flux-tunable transmons (tuned using SQUID non-linear inductors and nearby flux lines) coupled to individual readout resonators with a common feedline, as well as the tunable coupler between them. This system is isolated from the rest of the chip by parking neighbouring qubits far away via external DC sources. See the figure on the right.
+
+The qubits are driven with analog IQ signal output pairs generated by the OPX+ and up-converted with an Octave. Their individual fluxes, as well as the flux of the tunable coupler, are tuned using three additional analog outputs from the OPX+.
+
+The readout resonators are driven with an analog IF signals pair generated by the OPX+ and up-converted with an Octave.
```

Comment:
We can add, 
"The signal from the fridge is returned to the Octave, where it's downconverted and then measured by the OPX."